<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alt::SettingsWidget</class>
 <widget class="QWidget" name="alt::SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="redirectionalSettingsTitleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>System Settings</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="applicationSettingsVerticalLayout">
     <item>
      <widget class="QCheckBox" name="startupCheckBox">
       <property name="text">
        <string>Do not run at the start of next sessions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="hardwareCheckBox">
       <property name="text">
        <string>Display hardware tab (Experimental)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="applicationSettingsTitleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Application Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="redirectionalSettingsVerticalLayout">
     <item>
      <widget class="QPushButton" name="runAccPushButton">
       <property name="text">
        <string>Run System management center</string>
       </property>
       <property name="icon">
        <iconset theme="alterator"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runCommonSettingsPushButton">
       <property name="text">
        <string>Run common settings</string>
       </property>
       <property name="icon">
        <iconset theme="preferences"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
